downstream/modules/platform/con-edge-manager-os-img-script.adoc

```diff
@@ -0,0 +1,9 @@
+[id="edge-manager-os-img-script"]
+
+= Operating system images with scripts
+
+Avoid executing scripts or commands that change the file system.
+The `bootc` or the {RedHatEdge} can overwrite the changed files that might cause a deviation or failed integrity checks.
```

Comment:
Suggest changing to: The `bootc` or the {RedHatEdge} can overwrite the changed files which can cause a deviation or failed integrity checks.